src/harness/testcases/WINNF_FT_S_WDB_testcase.py

```diff
@@ -160,11 +187,11 @@ def generate_WDB_2_default_config(self, filename):
     cpi_name_d = 'd_name'
 
     # Read private keys for the CPI users
-    with open(os.path.join('testcases', 'testdata', 'wdb_2', 'WDB_2_CPI_Private_Key.txt'),
+    with open(os.path.join('testcases', 'testdata','WDB_2_CPI_Private_Key.txt'),
```

Comment:
(1) You should be generating the public+private key pair in this file (see the REG test cases for an example, it's a single function call), not reading them from disk.
(2) You need write out only the public key, NOT the private key.
(3) Line 194 uses the same file. The CPIs need to have distinct keys.
(4) As you did above, please save the filename in a local var and then use that directly in the config.

Reply:
Fixed as per suggestion.